apple_os/auditing/configure_auditing/audit_failure_halt/bash/shared.sh

```diff
@@ -0,0 +1,7 @@
+# platform = Apple macOS 10.15
+# reboot = false
+# strategy = enable
+# complexity = low
+# disruption = low
+
+/usr/bin/sed -i.bak '/^policy/ s/$/,ahlt/' /etc/security/audit_control; sudo /usr/sbin/audit -s
```

Comment:
Was ``ahlt`` meant to be ``halt``?

Reply:
It's `ahlt`. I did a double take on it as well.